internal/xds/translator/ratelimit.go

```diff
@@ -179,7 +190,7 @@ func buildRouteRateLimits(descriptorPrefix string, global *ir.GlobalRateLimit) [
 						},
 					},
 				}
-				rlActions = append(rlActions, action)
+				rlActions = append(rlActions, routeDescriptor, action)
```

Comment:
Are we appending the `routeDescriptor` multiple times in the for loop ?

Reply:
yes. the `routeDescriptor` is also a `RateLimit_Action` type, should be append before each Header/CIDR match action:
https://github.com/envoyproxy/gateway/blob/6ec8afd85733b61daf0df8dc2aa10a9fb5da5239/internal/xds/translator/ratelimit.go#L151